.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,14 @@
+# Proposed Changes
+
+<!--
 Hi there. Thanks for submitting a pull request!
 
 Please describe your changes in this PR description.
 You can also include a link to an active Jira ticket that provides relevant context if there is one.
 
-https://policies.zephyrai.bio/sdlc.html
+https://zephyrai.atlassian.net/wiki/spaces/HELIX/pages/37650458/Secure+Software+Development+and+Product+Security
+-->
 
-# Proposed Changes
+**Jira ticket (optional)**: https://zephyrai.atlassian.net/browse/TICKET
```

Comment:
do we need `optional` here? If it really is optional you can just delete the line

Reply:
It's probably redundant, but I like that it's explicit.